cache/remotecache/v1/spec.go

```diff
@@ -20,10 +20,11 @@ type CacheLayer struct {
 }
 
 type LayerAnnotations struct {
-	MediaType string        `json:"mediaType,omitempty"`
-	DiffID    digest.Digest `json:"diffID,omitempty"`
-	Size      int64         `json:"size,omitempty"`
-	CreatedAt time.Time     `json:"createdAt,omitempty"`
+	MediaType  string            `json:"mediaType,omitempty"`
+	DiffID     digest.Digest     `json:"diffID,omitempty"`
+	Size       int64             `json:"size,omitempty"`
+	CreatedAt  time.Time         `json:"createdAt,omitempty"`
+	Additional map[string]string `json:"additional,omitempty"`
```

Comment:
I don't have memory why this structure was called annotations, defined as a struct, but "additional" looks quite weird here.

I think the options here are either:
- Find what extra annotations are needed here and add it as extra fields.
- Make `Additional/Rest` `json:"-"` and add a custom marshal/unmarshal to `LayerAnnotations` so that it still creates a single key-value pairs object in JSON.